File: baekjoon/1000/1076.cpp
#include<iostream>
#include <math.h>
using namespace std;

long long Conv(string s)
{
    if (s == "black") return 0;
    else if (s == "brown") return 1;
    else if (s == "red") return 2;
    else if (s == "orange") return 3;
    else if (s == "yellow") return 4;
    else if (s == "green") return 5;
    else if (s == "blue") return 6;
    else if (s == "violet") return 7;
    else if (s == "grey") return 8;
    else return 9;
}

int main()
{
    cin.tie(NULL);
    std::ios::sync_with_stdio(false);
    long long a, b, c;
    string s;
    cin >> s;
    a = Conv(s);
    cin >> s;
    b = Conv(s);
    cin >> s;
    c = Conv(s);
    a = (10 * a + b) * pow(10, c);
    cout << a;
    return 0;
}

File: baekjoon/1000/1076.rs
use std::io::*;

struct Scanner {
    it: std::str::SplitAsciiWhitespace<'static>,
}

impl Scanner {
    fn new() -> Self {
        let mut s = String::new();
        stdin().read_to_string(&mut s).ok();
        Self { it: s.leak().split_ascii_whitespace() }
    }
    fn raw(&mut self) -> &str {
        self.it.next().unwrap()
    }
}

fn main() {
    let mut sc = Scanner::new();
    let mut bw = BufWriter::new(stdout().lock());
    let conv = |s: &str| -> u64 {
        match s {
            "black" => 0,
            "brown" => 1,
            "red" => 2,
            "orange" => 3,
            "yellow" => 4,
            "green" => 5,
            "blue" => 6,
            "violet" => 7,
            "grey" => 8,
            "white" => 9,
            _ => unreachable!(),
        }
    };
    let x = conv(sc.raw());
    let y = conv(sc.raw());
    let z = conv(sc.raw());
    let ans = (10 * x + y) * 10_u64.pow(z as u32);
    writeln!(bw, "{ans}").ok();
}
